internal/handlers/sqlite/msg_explain.go

```diff
@@ -83,6 +85,44 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		qp.Filter = params.Filter
 	}
 
+	if h.EnableSortPushdown {
```

Comment:
I think one more place to add this thing is `msg_aggregate` (see the old `msg_aggreagte` for pg).

Reply:
For msg_aggregate we don't support yet both filter and sort pushdown. It seems to me that in old pg handler the logic for deciding when to do pushdown and not was a bit more complex, so I'd suggest to implement those pushdowns in separate issue to hopefully merge this one before the weekend :pray:

Reply:
@noisersup let's reuse the issue #3520 then! could you describe there what needs to be done to support it in aggregation, please?

Reply:
Sure!